ballista/core/src/serde/generated/ballista.rs

```diff
@@ -1985,7 +1985,7 @@ pub mod executor_grpc_client {
 pub mod scheduler_grpc_server {
     #![allow(unused_variables, dead_code, missing_docs, clippy::let_unit_value)]
     use tonic::codegen::*;
-    /// Generated trait containing gRPC methods that should be implemented for use with SchedulerGrpcServer.
+    ///Generated trait containing gRPC methods that should be implemented for use with SchedulerGrpcServer.
```

Comment:
Is it necessary to remove the blank after the ///

Reply:
No, not sure why it was removed in the generated code.

Reply:
Can you remind me again why the generated code is checked in? Where did we land on that?

Reply:
I think it was so the project played nicely with IDEs (some of which do not evaluate proc macros)